.vuepress/config.js

```diff
@@ -79,11 +79,29 @@ module.exports = {
           collapsable: false,
           children: [
             '/learn/advanced/asynchronous_updates',
-            '/learn/advanced/snapshots_vs_dumps',
             {
               title: 'Updating MeiliSearch',
               path: '/learn/advanced/updating.md',
             },
+            {
+              title: 'Data backup',
+              collapsable: false,
+              sidebarDepth: 0,
+              children: [
+                {
+                  title: 'Dumps',
+                  path: '/learn/advanced/dumps',
+                },
+                {
+                  title: 'Snapshots',
+                  path: '/learn/advanced/snapshots',
+                },
```

Comment:
Unless I'm mistaken you've also got to remove these two from the `Feature references` sidebar.